docs/changelog.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+* **21 Jan 2019:** Added an upload configuration to the client worker.
+* **22 Jan 2019:** Upgraded worker SDK version to `13.5.1`.
+* **30 Jan 2019:** 
+  * Moved the snapshot file from `./shared`  to `./SpatialOS/snapshots`.
+  * Added this changelog.
+  * Added a known issues section to the [readme](../README.md).
```

Comment:
Could link directly to the known issues section instead of the readme?

Reply:
Great idea. Done!